src/targets/node/fetch.js

```diff
@@ -27,8 +27,8 @@ module.exports = function (source, options) {
     method: source.method
   }
 
-  if (Object.keys(source.headersObj).length) {
-    reqOpts.headers = source.headersObj
+  if (Object.keys(source.allHeaders).length) {
+    reqOpts.headers = source.allHeaders
```

Comment:
What are the implications of consuming `allHeaders` instead of `headersObj`?

Reply:
`allHeaders` contains the `cookie` header if it's present, `headersObj` is a k/v store of headers from the HAR excluding `cookie` because that's dynamically generated from the `cookies` array in the HAR.

Reply:
And by changing this to `allHeaders` this target no longer needs to combine its own cookie data into a `cookie` header because that's already been done in `src/index.js`.

Reply:
Love it, thanks for the explanation!